source/ProcessManager.Components/EnqueueActorMessages/EnqueueActorMessagesSyncClient.cs

```diff
@@ -12,16 +12,39 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System.Text;
+using System.Text.Json;
 using Energinet.DataHub.ProcessManager.Components.Abstractions.EnqueueActorMessages;
+using Energinet.DataHub.ProcessManager.Components.Extensions.DependencyInjection;
 
 namespace Energinet.DataHub.ProcessManager.Components.EnqueueActorMessages;
 
-internal class EnqueueActorMessagesSyncClient : IEnqueueActorMessagesSyncClient
+internal class EnqueueActorMessagesSyncClient(
```

Comment:
If we want to follow the naming we've used for the Process Manager client, i think the "sync/http" client have been called `ProcessManagerClient`, and the "async/servicebus" client is named `ProcessManagerMessageClient`. I'm not sure we want to rename it here, but i do think that `EnqueueActorMessagesHttpClient` is a better name than `EnqueueActorMessagesSyncClient`.

```suggestion
internal class EnqueueActorMessagesHttpClient(
```